astropy/io/fits/connect.py

```diff
@@ -180,6 +182,10 @@ def read_table_fits(
         string columns. Set this parameter to `False` to avoid the performance
         penalty of doing this masking step. The masking is always deactivated
         when using ``memmap=True`` (see above).
+    strip_spaces : bool, optional
```

Comment:
Related to the `whitespace` vs `spaces` issue, I wonder if this should be more descriptive like `strip_trailing_whitespace`. I am not adamant, but just putting out the idea.